[41기 류건호] Westagram Backend Project - Bcrypt를 이용하여 비밀번호 암호화하기

koreanMike513 opened the pull request:
## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [ ] 기능 추가
- [ ] 데이터베이스 작업
- [ ] 리팩토링
- [x] 버그 수정
- [ ] 컨벤션 수정

<br />

## :: 구현 목표 - 해당 브랜치(PR)에서 구현하고자 하는 하나의 목표 작성
- email, password를 이용한 로그인 기능 구현 (예시)

<br />

## :: 구현 사항 설명 - 해당 브랜치(PR)에서 작업한 내용 작성
회원가입 시 bcrypt를 통해 user password 암호화 

<br />

## :: 테스트 결과 이미지
1. Server가 잘 동작하는지 확인할 수 있는 Terminal 캡쳐 이미지
2. Postman(Client Tool)을 이용한 API 테스트 결과 이미지
3. DB 작업 PR인 경우 Table 생성 및 수정 결과에 대해 확인할 수 있는 이미지

![Assignment](https://user-images.githubusercontent.com/104866923/209515433-ec7b70e1-1288-4012-8f3b-76ec716ef613.png)

![스크린샷 2022-12-26 오후 4 03 54](https://user-images.githubusercontent.com/104866923/209515604-477bba60-e367-4109-8830-ea9dbd9ccf2c.png)


<br />

## :: 기타 질문 및 특이 사항
- 제가 작성한 함수의 로직이 너무 복잡하다고 생각이 됩니다. 더 효율적인 방법이 없을까요? (예시)

koreanMike513 committed:
[ADD]: 일단은 final

koreanMike513 committed:
[ADD]: Authentication 구현

SeongJoon-K reviewed (changes requested):
건호님! 승기님 디렉토리의 수정사항이 존재합니다. 해당 PR을 merge하게 되었을 때, 승기님이 작성하신 파일에 conflict가 발생할 수 있습니다. 해당 사항을 참고해서 Filechanged에 건호님 파일만 남을 수 있도록 다시 수정해 주세요!

koreanMike513 committed:
[FIX]: merge issue

koreanMike513 committed:
[CONFLICE]: fixed conflict

koreanMike513 committed:
[ADD]: fixed conflict

koreanMike513 committed:
"conflict fix"
Merge branch 'main' of https://github.com/wecode-bootcamp-korea/41-backend-westagram-1

koreanMike513 committed:
conflict fix

koreanMike513 committed:
Merge branch 'main' into feature/gunho-signup

SeongJoon-K commented:
건호님 현재 File changed에 다른 교육생 디렉토리 제거 이력이 존재합니다. 해당 수정사항이 발생하였을 때, merge하게 된다면, main branch에 필연적으로 conflict 가 발생합니다. 수정하시고 다시 push 해 주세요.

koreanMike513 committed:
[FIX]: conflict fix

koreanMike513 committed:
conflict

Merge branch 'main' into feature/gunho-signup

koreanMike513 committed:
[FIX]: conflict fix
